common/pref_names.cc

```diff
@@ -96,6 +96,12 @@ const char kMRUCyclingEnabled[] = "brave.mru_cycling_enabled";
 const char kTabsSearchShow[] = "brave.tabs_search_show";
 const char kDontAskForCrashReporting[] = "brave.dont_ask_for_crash_reporting";
 
+// Cast extension requires a browser restart once the setting is toggled.
+// kBraveMediaRouter is used as a proxy to identify the current state of the
+// switch and prefs::kEnableMediaRouter is updated to kBraveMediaRouter
+// on restart.
+const char kBraveMediaRouter[] = "brave.media_router";
```

Comment:
this naming is confusing imo. It makes it sound like we have our own media router. If this pref is needed (not sure why we can't just restart when changing `kEnableMediaRouter`) it should be renamed to something like `kEnableMediaRouterOnRestart` or something

Reply:
resolved